src/main/java/org/fundacionjala/coding/KevinHerrera/Movies/Customer/Customer.java

```diff
@@ -0,0 +1,111 @@
+package org.fundacionjala.coding.KevinHerrera.Movies.Customer;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.fundacionjala.coding.KevinHerrera.Movies.Movie.Movie;
+import org.fundacionjala.coding.KevinHerrera.Movies.Rental.RentalMovie;
+import org.fundacionjala.coding.KevinHerrera.Movies.Rental.RentalFactory;
+
+/**
+ * @author KevinHerrera - AWT-[01].
+ * @version 0.1
+ */
+public class Customer implements IRentMovie {
+    private String name;
+    private List<RentalMovie> rentedMoviesList;
+    private int frequentRenterPoints = 0;
+
+    /**
+     * .
+     * @param name name
+     */
+    public Customer(final String name) {
+        this.name = name;
+        rentedMoviesList = new ArrayList<RentalMovie>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)